neqo-transport/src/stats.rs

```diff
@@ -13,6 +13,8 @@ use std::fmt::{self, Debug};
 use std::ops::Deref;
 use std::rc::Rc;
 
+pub(crate) const MAX_PTO_COUNTS: usize = 10;
```

Comment:
Not sure if this can be fully public.  It's going to be visible via pto_counts, which is also public.  Note that this means we are committing to this value.

This works out being enough PTOs for a connection to hit an idle timeout at around a 16ms base PTO.  That seems like it might be high, but it's not very high.